packages/webpack/src/presets/base.ts

```diff
@@ -39,7 +39,9 @@ function baseConfig (ctx: WebpackConfigContext) {
       ...options.webpack.optimization,
       minimizer: []
     },
-    experiments: {},
+    experiments: {
+      ...options.webpack.experiments
```

Comment:
I think we have to handle the case where `false` is passed - I think this will throw an error in that case.

Reply:
tested it :D weird but 

```
{
...false
}
```

==> {}

Reply:
You are quite right - my mistake.